test/metabase/xrays/automagic_dashboards/core_test.clj

```diff
@@ -49,7 +52,7 @@
 
 ;;; ------------------- `->root source` -------------------
 
-(deftest ^:parallel source-root-table-test
+(deftest ^:paralell source-root-table-test
```

Comment:
These were originally spelled correctly, and now are broken.

(Tangentially, a linter for `^:paralell` and other typos might be handy?)